source/input.c

```diff
@@ -0,0 +1,68 @@
+#ifndef STRUCTS
+#define STRUCTS
+
+#include "../headers/structs.h"
+
+#endif
+
+#ifndef FINAL
+#define FINAL
```

Comment:
```suggestion
#define INPUT
```

See previous comment